masonry/src/event_loop_runner.rs

```diff
@@ -732,6 +732,17 @@ impl MasonryState<'_> {
                 render_root::RenderRootSignal::ShowWindowMenu(position) => {
                     window.show_window_menu(position);
                 }
+                render_root::RenderRootSignal::WidgetSelectedInInspector(widget_id) => {
+                    let (widget, state) = self.render_root.widget_arena.get_pair(widget_id);
+                    let widget_name = widget.item.short_type_name();
+                    let display_name = if let Some(debug_text) = widget.item.get_debug_text() {
+                        format!("{widget_name}<{debug_text}>")
```

Comment:
This is unfortunately sparse for a lot of widgets. I suppose that makes a good follow-up?